internal/cli/cmd/cli.go

```diff
@@ -145,6 +146,9 @@ A Command Line Interface for KubeBlocks`,
 			if cmd.Name() == cobra.ShellCompRequestCmd {
 				kcplugin.SetupPluginCompletion(cmd, args)
 			}
+			if cmd.Name() != "login" && !auth.IsLoggedIn() {
```

Comment:
Not all commands require login. We need to maintain a whitelist for the commands that need to execute login. Only the commands in it need to prompt the user to execute login first.

Reply:
LGTM